README.md

```diff
@@ -1,50 +1,31 @@
 [![Sourcery](https://img.shields.io/badge/Sourcery-enabled-brightgreen)](https://sourcery.ai)
 
-## Sourcery
-Instantly review and improve your Python code to keep your codebase maintainable. 
+## Sourcery Code Reviews
+Sourcery will review any pull request on any GitHub repository to provide instant feedback on the proposed changes.
```

Comment:
**🔨 issue:** This section doesn't explain why I should install it

Right now it is an explanation of what Sourcery is. Lead with the value propositions:
- Spend less time reviewing code
- Ensure high quality code
- Automated approvals
- Accelerate your velocity